.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: python=${{ matrix.python }} os=${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
It looks like backoffice is running ubuntu 18.04. What would you think about adding that into the matrix here?

Reply:
Sure, that's reasonable. I don't expect this OS version difference to impact our test results very much, but it's definitely possible. I've pushed an additional commit here and on https://github.com/seattleflu/id3c-customizations/pull/211.

Reply:
Right on. Thanks Thomas!